src/main/java/com/celements/web/classcollections/oldcore/IOldCoreClassDef.java

```diff
@@ -0,0 +1,18 @@
+package com.celements.web.classcollections.oldcore;
```

Comment:
up until now we have used the package name "classes" for the new class definitions. perhaps rename the package to **com.celements.web.classes**